herddb-core/src/test/java/herddb/core/UpdateTest.java

```diff
@@ -244,7 +244,8 @@ public void upsertTest() throws Exception {
             }
 
             // upsert, making n1 null now, because it has not been named in the INSERT clause
-            assertEquals(1, executeUpdate(manager, "UPSERT INTO tblspace1.tsql(k1,s1) values(?,'non-empty')", Arrays.asList("mykey")).getUpdateCount());
+            // use non uppercase casing in UPSERT keyword
+            assertEquals(1, executeUpdate(manager, "UPsert INTO tblspace1.tsql(k1,s1) values(?,'non-empty')", Arrays.asList("mykey")).getUpdateCount());
```

Comment:
Is calcite able to support mix case characters now?

Reply:
Yes it always did.
We had problems only on the jsqlparser part